models/db/api-key.model.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
+  const ApiKey = sequelize.define('Api_Key', {
```

Comment:
The string `'Api_Key'` should be `'ApiKey'`. This is the value index.js uses to create the class name of the model.